recipes/glymur/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.8.3" %}
+package:
+  name: glymur
+  version: {{ version }}
+
+source:
+  fn: Glymur-0.8.3.tar.gz
+  url: https://pypi.python.org/packages/33/6e/3bb303c46e2b9a795389115401fd32f1eff16c12a66c7a53fa36c244f9bc/Glymur-0.8.3.tar.gz
+  md5: 2ff4c70a63a1e01f84b18e0ec4eb9b88
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - jp2dump=glymur.command_line:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.7.0
+    - lxml >=2.3.2
+    - mock
```

Comment:
Just for tests.  I should remove that...

On Fri, May 13, 2016 at 11:59 AM, Filipe <notifications@github.com> wrote:

> In recipes/glymur/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/605#discussion_r63207713>
> :
>
> > +  url: https://pypi.python.org/packages/33/6e/3bb303c46e2b9a795389115401fd32f1eff16c12a66c7a53fa36c244f9bc/Glymur-0.8.3.tar.gz
> > +  md5: 2ff4c70a63a1e01f84b18e0ec4eb9b88
> > +
> > +build:
> > +  number: 0
> > +  script: python setup.py install --single-version-externally-managed --record record.txt
> > +  entry_points:
> > +    - jp2dump=glymur.command_line:main
> > +
> > +requirements:
> > +  build:
> > +    - python
> > +    - setuptools
> > +    - numpy >=1.7.0
> > +    - lxml >=2.3.2
> > +    - mock
>
> Is mock a build and run dependency? Or is it just for tests?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/605/files/d4229db6df4a8530c5778ff1fba2a2fc872d612a#r63207713>
>



-- 
John Evans